migrations/versions/rev-2021-01-20-23:37:40-a87d9f892893_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: a87d9f892893
+Revises: 9afc35c9f00f
+Create Date: 2021-01-20 23:37:40.235040
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'a87d9f892893'
+down_revision = '9afc35c9f00f'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('ticket_holders', sa.Column('accept_receive_emails', existing_type=sa.Boolean(), nullable=True))
+    op.alter_column('ticket_holders', sa.Column('accept_share_details', existing_type=sa.Boolean(), nullable=True))
+    op.alter_column('ticket_holders', sa.Column('accept_video_recording', existing_type=sa.Boolean(), nullable=True))
```

Comment:
What's changed? "Accept Video Recording" is still "Accept Video Recording" in the DB